docs/getting-started/first-service.md

```diff
@@ -19,7 +19,7 @@ Let's get you started on creating your first service using our service creation
 
 ## Prerequisites
 
-First, [sign up](https://app.amplication.com/login) for an Amplication account. Next, authorize with GitHub.
+First, [sign up](https://app.amplication.com/login) for an Amplication account with GitHub or your email address.
```

Comment:
@dericksozo - Maybe we should emphasize "or your business email address" to exclude gmail and such. WDYT?

Reply:
Hey @PazYanoverr,  I got this one in the latest commit.